src/DebugEngineHost.Stub/DebugEngineHost.Stub.csproj

```diff
@@ -20,7 +20,7 @@
     of DebugEngineHost, so we don't want to conflict with the real version. -->
     <OutputPath>bin\$(Configuration)</OutputPath>
     <DocumentationFile>$(DropRootDir)\ReferenceAssemblies\$(AssemblyName).xml</DocumentationFile>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
>net472 [](http://example.com/codeflow?start=21&length=6)

You mentioned that it was difficult to stay on net462 because of the threading analyzer. Is that being used on any of the projects that we want to use with VS Code?

Reply:
Ah, the issue might be that we are loading Analyzers for all projects via https://github.com/microsoft/MIEngine/blob/main/build/Analyzers.targets